src/odemis/gui/cont/tabs/cryo_chamber_tab.py

```diff
@@ -90,8 +90,9 @@ def __init__(self, name, button, panel, main_frame, main_data):
         self.txt_projectpath = self.panel.txt_projectpath
         self.btn_load_project.Hide()
 
-        # Create new project directory on starting the GUI
-        self._create_new_dir()
+        # indicates whether a project has been initially created or loaded
+        self._is_initial_project_ready: bool = False
```

Comment:
I don't really get what this means. Does this mean that the data is explicitly stored in a project selected by the user (in contrast to the data being stored in the default new project folder created at init)?
Could you clarify this in the comment (or the variable name).

Reply:
It's a flag to indicate whether the user has initially created a new project or loaded an existing one. The rest of the tab is blocked until they succesffuly do that.